pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdGenerateDocument.java

```diff
@@ -107,9 +107,6 @@ private void generateDocument(StringBuilder sb, String module, JCommander obj) {
                     .append("\n```");
             sb.append("\n\n");
             CmdBase cmdObj = (CmdBase) obj.getObjects().get(0);
-            for (String s : cmdObj.jcommander.getCommands().keySet()) {
-                sb.append("* `").append(s).append("`\n");
-            }
```

Comment:
Could you share a preview of diff how these lines affect the final result?

Reply:
![image](https://user-images.githubusercontent.com/32540679/196902992-874c0e0c-498b-4728-b65d-24bf7a6ef5bb.png)

These lists are removed by removing these code. They are redundant because we now already have a TOC on the right side.